README.md

```diff
@@ -329,6 +329,18 @@ response.user.show(1)
 #=> HateoasResponse
 ```
 
+### Circuit Breaker
+
+The client ships with a circuit breaker for get requests, this can be enabled by setting `ENABLE_API_CLIENT_CIRCUIT`. The Circuit Breaker is powered by [Circuitbox](https://github.com/yammer/circuitbox).
+
+The following parameters can be used:
+
+- `CIRCUIT_BREAKER_SLEEP_WINDOW` - The time in seconds the circuit will remain open once it's triggered
```

Comment:
same comment as above for these. also, the timeout config var isn't documented here.